docs/devguide/xtypes.rst

```diff
@@ -821,6 +821,24 @@ Any enumerators that are not annotated take on the integer value one higher than
     @value(2) TWO
   };
 
+
+Marking member as optional
+==========================
+
+.. _xtypes--optional:
+
+``@optional``
+-------------
+
+..
+    Sect<16.6.8.1>
+
+Applies to: struct members (only when :ref:`opendds_idl--using-the-idl-to-c-11-mapping`)
+
+This annotation marks a member as optional. In the IDL-to-C++11 mapping it generates either an std::optional<T> or an equivalent custom implementation of std::optional.
+
+More information about other language bindings for @optional can be found [here](https://github.com/OpenDDS/OpenDDS/wiki/Proposal:-@optional-support)
```

Comment:
```suggestion
More information about other language bindings for ``@optional`` can be found `here <https://github.com/OpenDDS/OpenDDS/wiki/Proposal:-@optional-support>`__
```

Markdown links won't work in reStructuredText. I'm not sure this line should be here or not.

Reply:
I can remove the link but, I added it because of the information with the classic C++ and Java bindings (relevant for this repo) could be useful for those wondering why they aren't supported and the plans for them